www/src/pages/contributing/stub-list.js

```diff
@@ -48,13 +68,24 @@ class StubListRoute extends React.Component {
               {` `}
               to learn more.
             </p>
-            <ul data-testid="list-of-stubs">
-              {stubs.map(stub => (
-                <li key={stub.title}>
-                  <Link to={stub.link}>{stub.title.slice(0, -1)}</Link>
-                </li>
-              ))}
-            </ul>
+            <section data-testid="list-of-stubs">
+              {sortedCategories.map(category => {
+                let categoryTitle =
+                  category.slice(-1) === `*` ? category.slice(0, -1) : category
```

Comment:
categories themselves may be stubs so remove the asterisk if it is there. Is this alright or should we clean this up somewhere else?